Library/Formula/decklinksdk.rb

```diff
@@ -0,0 +1,20 @@
+require 'formula'
+
+class Decklinksdk < Formula
+  homepage 'http://www.blackmagicdesign.com/support/sdks/'
+  url 'http://software.blackmagicdesign.com/SDK/Blackmagic_DeckLink_SDK_10.1.4.zip'
```

Comment:
As @mistydemeo suggested, I removed steps to install the proof-of-concept binaries in this commit d935d34e44f7e77af103e81adf2dfa0f7fd8a6a7. To @mikemcquaid, I'm not sure that the libraries of Mac/include can be built from what is provided.